### DISCUSSION TITLE
transitive peerings in route table

### DISCUSSION THREAD

**Discussion by michael-novak-80** [Azure/avdaccelerator#126] [Q&A]
Hello, 
We are testing AVD accelerator for our deployments. We have deployed hub and spoke network topology into a new (test subscription), through bicep template. We have three networks (Azure AD DS, Hub Network (with Virtual Gateway), and AVD resources network). Peerings from Azure ADDS, AVD resources networks are configured to and from Hub Network.

Our issue is that the accelerator creates an empty route table in rg-westeurope-avd-app1-network resource group. Unless we configure transitive peering i.e. define custom UDRs in this route table (and assign them to respective subnets between AVD Resources network and Azure AD DS network) AVD session hosts cannot reach Azure ADDS. 

Isn't this something the accelerator should do automatically? 

Thanks.

**Comment by danycontre** [comment]
@michael-novak-80 missed this discussion, please let us know if this request is still valid.
